src/components/exportAndImport/importFromJson.ts

```diff
@@ -61,6 +61,7 @@ const constructInsertAndGetter = (
   columns.forEach(({ isPrimary, isGenerated, columnName }) => {
     if (!preserveIds && isPrimary) return
     if (isGenerated) return
+    if (values[columnName] === undefined) return
```

Comment:
Can this be:
```
if (values?.[columnName]) return
```
That would cover "undefined", "Null", empty strings, and just missing.

Reply:
I thought it's good to differentiate between null or empty string to undefined (since those values would have been exported and perhaps should be imported), vs not exported fields:values which (for columns that were added since export)